<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	border-image: url(:/new/pics/pic (1).jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 170, 127);
font:20px &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Random Sequence 6.1.0 beta</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>391</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:1px solid white;
font:20px &quot;Lucida Sans&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font:15px &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>[Open File]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font:15px &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>[Open File]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: grey;
font:15px &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>[Open File]</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>391</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
     </property>
     <property name="text">
      <string>Open(&amp;O)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
     </property>
     <property name="text">
      <string>Stop(&amp;P)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
     </property>
     <property name="text">
      <string>Up(&amp;U)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
     </property>
     <property name="text">
      <string>Down(&amp;D)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
     </property>
     <property name="text">
      <string>Generate(&amp;G)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
     </property>
     <property name="text">
      <string>Add(&amp;A)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
     </property>
     <property name="text">
      <string>Save(&amp;S)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
     </property>
     <property name="text">
      <string>Save As(&amp;K)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
     </property>
     <property name="text">
      <string>Delete(&amp;D)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_17">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
     </property>
     <property name="text">
      <string>Clear(&amp;L)</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:30px &quot;Lucida Console&quot;;</string>
    </property>
    <property name="text">
     <string>Tools:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>340</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>340</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font:15px &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font:15px &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>370</y>
      <width>211</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
    </property>
    <property name="text">
     <string>Generate Number Sequence</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font:15px &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>340</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>440</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame{
border:0px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>171</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border:1px solid white;
font:20px &quot;Lucida Sans&quot;;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
     </property>
     <property name="text">
      <string>Add(&amp;A)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
     </property>
     <property name="text">
      <string>Delete(&amp;D)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
     </property>
     <property name="text">
      <string>Up(&amp;U)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_16">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
     </property>
     <property name="text">
      <string>Down(&amp;D)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:15px &quot;Lucida Sans&quot;;</string>
     </property>
     <property name="text">
      <string>Use Hidden Sequence</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>410</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
    </property>
    <property name="text">
     <string>View Hidden Sequence</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:0px;
	background-color: rgb(0, 170, 127);
	color: rgb(255, 255, 255);
	font:15px &quot;Lucida Sans&quot;;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
    </property>
    <property name="text">
     <string>Clear History</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:15px &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Do not repeat</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_2</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>732</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
